tests/vnc_test.go

```diff
@@ -346,8 +352,16 @@ func (h ResolutionMatcher) NegatedFailureMessage(actual interface{}) (message st
 func getResolution(domain *launcherApi.DomainSpec) (X, Y int) {
 	videoType := domain.Devices.Video[0].Model.Type
 	if videoType == "virtio" {
-		X = 1280
-		Y = 800
+		arch := util.TranslateBuildArch()
+		// Use different resolution for s390x.
```

Comment:
Specifically because the VGA device provided by QEMU on s390x only supports this resolution? Can you document the answer here please.

Reply:
review comment not applicable after reverting the changes.